CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -0,0 +1,718 @@
+with more than one payload at a time.
+
+==Piercing Spike ==
+Adrenaline Cost:35 Points
+Needle Cost: 1 Needle
+Duration: 	2 Actions
+Range:		45m
+    2 chance to miss. Deal 42 piercing damage. AP level 5. If the  needle deals 
+damage to the target's health and the target is still alive, apply a poison 
+payload of your choice. 
+    
+    At level 10, has AP level 6 instead. At level 15, has AP level 8 and deals 
+65 damage. At level 20, ignores physical armor and deals 80 damage.
+
+==Poison: Mild Hallucinogen ==
+Adrenaline Cost:20 Points
+Contagion Cost: 1 Contagion
+Duration: 	5 turns
+Range:		Payload (use in combination with other powers)
+    Target must pass a DC 70 Will check. If they fail, they suffer one of four 
+effects as chosen by GM or by random dice roll listed below. Hallucinogens only 
+affects biologic targets.
+- Target panics and runs away as fast as he can
+- Target believes nearest ally is a vicious, man-eating, submarine-sized, ferret bent on eating his belly button and attacks accordingly
+- Target is tricked into thinking he is being eaten alive by bugs and strips naked immediately and attempts to squish them all
+- Target discovers that their weapon is actually a sprinkler, and begins watering the grass diligently.
+
+==Poison: Depressant ==
+Adrenaline Cost:30 Points
+Contagion Cost: 1 Contagion
+Duration: 	engagement
+Range:		Payload (use in combination with other powers)
+    Target must pass a DC 70 Will check. If they fail, target has their speed 
+reduced to 25%, round down and takes a +4 to miss. Depressant always deploys as 
+an area of effect cloud, even with deployment powers that otherwise wouldn't. 
++5m radius area of effect. Depressant only affects biologic targets. Subsequent
+doses of depressant increase the DC by a stacking 10 points each, but the 
+negative effects of succumbing to the depressant effect do not stack.
+
+== Contaminate Area ==
+Adrenaline Cost: 50 Points
+Contagion Cost: 1 Contagion
+Duration: 2 Actions
+Range: STR m
+    The Doctor sprays out a noxious contaminant onto a 2m x 4m rectangle with 
+its nearest corner within STR (not strength mod) meters from the Doctor. The 
+Doctor must be able to see this area. Biologic targets without gas masks that
+are in the affected area when the Doctor sprays it or that end their turn in the
+area for 5 turns after it is sprayed must make a FORT save. The FORT save has a
+DC of 110. If a target succeeds on the save the area does not affect them and
+they do not need to save again if they are still in the area on subsequent 
+turns. If they fail the save, they are "contaminated." Races that are extra
+sensitive to smoke or disease have disadvantage on this save. At level 10, this
+area expands to up to 3m x 5m at the discretion of the Doctor.
+
+    "Contaminated" targets get a -2 to melee or weapon accuracy rolls, and lose
+5 health and 5 nanites at the end of their turn every turn until Contamination
```

Comment:
Idk if we have rules on it, but what happens when the target has 0 Nanites in their Nanite pool? Or what if they had 3 Nanites and need to lose 5?